spec.md

```diff
@@ -419,25 +421,7 @@ the set of known extensions - not just for possible name conflicts but for
 extensions that might be of interest.
 
 Each specification that defines how to serialize a CloudEvent will define how
-extension attributes will appear.
-
-Extension attribtue MUST be serialized using the same general pattern as all
-CloudEvents context attributes. For example, in binary HTTP, that means they
-MUST appear as HTTP headers with the `ce-` prefix. The specification of an
-attribute MAY define a secondary serialization where the data is duplicated
-in some other location within the message.
-
-In cases where a secondary serialization is defined, the extension
-specification MUST also state what a receiver of the CloudEvent is to do
```

Comment:
Are we losing something by dropping this MUST?  Granted it's more of a statement for spec authors instead of people writing the code, but isn't the guidance/requirement needed to ensure interop ?

Reply:
IMO no. This MUST belongs in binding specs that introduce such notions as "alternative locations" - e.g. it's there in the  AMQP spec.  In the core spec it's confusing and unhelpful speculation about the possible detailed inner workings of some bindings, and entirely irrelevant and meaningless for others (e.g. JSON format, MQTT 3.1 etc.)

Reply:
Not sure I agree since then each protocol spec would need to duplicate the same type of text when it's generic text that isn't protocol specific.  However, even so, this PR doesn't add the text to the protocol docs so wouldn't more work be needed before we dropped the text here?

Reply:
Nope. If you want a MUST in the core it should say "binding specs MUST be clear and unambiguous enough that different implementations of the binding spec will inter-operate" - it should not say how particular bindings might or might not accomplish that. However I think that's already implied by the definition of a binding spec.

Reply:
See latest